clouddriver-kubernetes/src/integration/java/com/netflix/spinnaker/clouddriver/kubernetes/it/DeployManifestIT.java

```diff
@@ -1246,4 +1247,199 @@ public void shouldBindRequiredCronJobDockerImage() throws IOException, Interrupt
+    final List<Map<String, Object>> request =
+        KubeTestUtils.loadJson("classpath:requests/deploy_manifest.json")
+            .withValue("deployManifest.account", ACCOUNT1_NAME)
+            .withValue("deployManifest.moniker.app", APP1_NAME)
+            .withValue("deployManifest.manifests", manifest)
+            .asList();
+    final List<String> deployments = KubeTestUtils.sendOperation(baseUrl(), request, "");
+    // ------------------------- then --------------------------
+    String exits = kubeCluster.execKubectl(String.format("get crd %s", crdName));
+    assertTrue(
+        deployments.size() == 1
+            && deployments.get(0).equals(String.format("customResourceDefinition %s", crdName))
+            && exits.contains(crdName));
+  }
+
+  @DisplayName(
+      ".\n===\n"
+          + "Given a CRD namespaced scope version v1\n"
+          + " And it's associated CR manifests"
+          + "When sending deploy manifests request\n"
+          + "Then a CRD and it's CR is created at namespaced level\n===")
+  @Test
+  public void shouldDeployCrCrdNamespacedV1() throws IOException, InterruptedException {
+    // ------------------------- given --------------------------
+    final String crdName = "crontabs.stable.example.com";
+    final String crName = "my-new-cron-object";
+    final List<Map<String, Object>> crdManifest =
+        KubeTestUtils.loadYaml("classpath:manifests/crd_v1.yml")
+            .withValue("metadata.name", crdName)
+            .withValue("spec.scope", "Namespaced")
+            .asList();
+    final List<Map<String, Object>> crManifest =
+        KubeTestUtils.loadYaml("classpath:manifests/cr_v1.yml")
+            .withValue("metadata.name", crName)
+            .asList();
+    // ------------------------- when --------------------------
+    final List<Map<String, Object>> crdRequest =
+        KubeTestUtils.loadJson("classpath:requests/deploy_manifest.json")
+            .withValue("deployManifest.account", ACCOUNT1_NAME)
+            .withValue("deployManifest.moniker.app", APP1_NAME)
+            .withValue("deployManifest.manifests", crdManifest)
+            .asList();
+    final List<Map<String, Object>> crRequest =
+        KubeTestUtils.loadJson("classpath:requests/deploy_manifest.json")
+            .withValue("deployManifest.account", ACCOUNT1_NAME)
+            .withValue("deployManifest.moniker.app", APP1_NAME)
+            .withValue("deployManifest.manifests", crManifest)
+            .asList();
+    final List<String> crdDeploymets = KubeTestUtils.sendOperation(baseUrl(), crdRequest, "");
+    final List<String> crDeploymets = KubeTestUtils.sendOperation(baseUrl(), crRequest, "default");
+    // ------------------------- then --------------------------
+    String crdExists = kubeCluster.execKubectl(String.format("get crd %s", crdName));
+    String crExists = kubeCluster.execKubectl(String.format("get %s %s", crdName, crName));
```

Comment:
Also here you can do `kubectl -n <the namespace> get <the crd> <the name>` to query in the explicit namespace where the CR was deployed.